homework/oop/homework_2.py

```diff
@@ -0,0 +1,29 @@
+# Home task 2
+# Implement method overriding in a test automation context.
+# Override a method in the child test class to customize the test execution.
+
+class ExampleTestCase:
+    def setUp(self):
+        print("Setup complete")
+
+    def runTest(self):
+        print("Run")
+
+    def tearDown(self):
+        print("Cleanup complete")
+
+
+class CustomTestExacution(ExampleTestCase):
+    def run(self):
```

Comment:
this method has a different name, so it extends the parent class instead of overriding it

Reply:
also, there is a typo in the class name